.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ parameters:
     default: "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com"
   workbench-image:
     type: string
-    default: "allofustest/workbench:buildimage-0.0.22"
+    default: "allofustest/workbench:buildimage-0.0.23"
   db-image:
     type: string
     default: "circleci/mysql:5.7"
```

Comment:
Looks like this can also be updated to cimg/mysql:5.7

Reply:
good catch, done